Resources/views/dashboard.html.twig

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Messenger monitor dashboard</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+</head>
+<body>
+<h1>Messenger monitor dashboard</h1>
+<div class="container">
+
+    {% for label, messages in app.flashes %}
```

Comment:
You should set a specific flash key and fetch out that specific key here (so we don't accidentally print out other, user-added flashes)

Reply:
sure! nice trick